src/content/popup/popup.ts

```diff
@@ -123,9 +125,15 @@ export function renderPopup(
     );
 
     windowElem.dataset.tabSide = options.tabDisplay || 'top';
-  }
 
-  const contentContainer = html('div', { class: 'content' });
+    console.log('Hello1');
```

Comment:
Can we drop this? 😅

```suggestion
```

Reply:
Lol yeah, sorry that slipped through from some debugging. Thanks for getting this merged, appreciate the patience with the back and forth!

Reply:
Not at all! Thank you so much for your patience with all my nitpicking!
I'll probably try to get in a few more fixes and then we can release the next version with this included.